Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean

```diff
@@ -1512,6 +1515,11 @@ lemma zpow_right_injective₀ (ha₀ : 0 < a) (ha₁ : a ≠ 1) : Injective fun
 @[simp] lemma zpow_right_inj₀ (ha₀ : 0 < a) (ha₁ : a ≠ 1) : a ^ m = a ^ n ↔ m = n :=
   (zpow_right_injective₀ ha₀ ha₁).eq_iff
 
+lemma zpow_eq_one₀ (ha₀ : 0 ≤ a)(ha₁ : a ≠ 1) {n : ℤ} : a ^ n = 1 ↔ n = 0 := by
```

Comment:
I didn't find a `pow` version. Could you add it?

Reply:
can we postpone this to a later PR as I think there are more lemmas missing for a `pow` version (at least if I try to do the proof in analogy with this one)